ci/playbooks/e2e-kuttl.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
We can also call 01-bootstrap.yml playbook. It will generate the make_install_yamls role.
```
- name: Bootstrap step
  ansible.builtin.import_playbook: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/playbooks/01-bootstrap.yml"
```